sessionctx/variable/noop.go

```diff
@@ -75,7 +75,7 @@ var noopSysVars = []*SysVar{
 	{Scope: ScopeNone, Name: "ft_max_word_len", Value: "84"},
 	{Scope: ScopeGlobal, Name: "log_backward_compatible_user_definitions", Value: ""},
 	{Scope: ScopeNone, Name: "lc_messages_dir", Value: "/usr/local/mysql-5.6.25-osx10.8-x86_64/share/"},
-	{Scope: ScopeGlobal, Name: "ft_boolean_syntax", Value: "+ -><()~*:\"\"&|"},
+	{Scope: ScopeGlobal, Name: "ft_boolean_syntax", Value: `+ -><()~*:""&|`},
```

Comment:
There are 2 bugs here. first the value differed from what the MySQL default is. Second, this value was not imported correctly in bootstrapping because there was no value escaping.